packages/components/src/Menu/Menu.tsx

```diff
@@ -154,6 +158,9 @@ export function Menu({
     whileElementsMounted: autoUpdate,
   });
 
+  const dismiss = useDismiss(context);
```

Comment:
missed this originally when moving Menu over

having Menu inside Modal, closing Menu with ESC would close the Modal too

this fixes it, so no more `useOnKeyDown` for ESC needed